packages/cli/bin/cli.js

```diff
@@ -82,12 +82,21 @@ const argv = yargs
     type: 'boolean',
   })
   .check(argv => {
-    if (getIsInProject(argv.src)) {
-      throw new Error(
-        i18n(`${i18nKey}.srcIsProject`, {
-          src: argv.src,
-        })
-      );
+    if (
+      argv._.length === 1 &&
+      (argv._.join(' ') === 'upload' || argv._.join(' ') === 'watch')
```

Comment:
Is `join` necessary here? I think this could be shortened to `['upload', 'watch'].includes(argv._[0])`